internal/catalogmetadata/client/client.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/operator-framework/operator-controller/internal/catalogmetadata"
 )
 
+type CatalogClient interface {
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#interfaces

Reply:
Updated; this interface is now declared in the `variablesources` pkg so we can create mocks for testing there.